src/CommonLib/hostfxr_utility.cpp

```diff
@@ -148,6 +178,16 @@ HOSTFXR_UTILITY::GetHostFxrParameters(
             {
                 hr = GetLastError();
```

Comment:
GetLastError() does not match to Hresult. You need to convert to with HRESULT_FROM_WIN32(GetLastError())